spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -2672,6 +2672,7 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
 
         Seq(true, false).foreach { ansiEnabled =>
           withSQLConf(
+            CometConf.COMET_LOG_FALLBACK_REASONS.key -> "true",
```

Comment:
is it related to this PR?

Reply:
no, this was for debugging - I have removed this now